.gitignore

```diff
@@ -1,4 +1,5 @@
 *.class
+*.classpath
```

Comment:
trying to gitignore the `.classpath` file, but somehow it shows up on my git diffs ...